build/common.sh

```diff
@@ -203,6 +207,18 @@ function kube::build::is_osx() {
   [[ "$(uname)" == "Darwin" ]]
 }
 
+function kube::build::update_dockerfile() {
+  if kube::build::is_osx; then
```

Comment:
Another option is just to remove `sed -i` and find a place for an outfile, then do a mv. That's the old POSIX way, before `-i` existed.